### ISSUE TITLE
Store a file temporarily 

### ISSUE THREAD

**Issue by katerosenthal** [cs481-ekh/f22-kneed-for-speed#112]
As a developer, I want to be able to store and access the data in a file temporarily, so that I can use the data in the file without needing permanent storage like a database.

## Definition of Done
- [ ] File is successfully stored in session or temp data
- [ ] File can be read and manipulated for use in app

## Tasks
- [ ] Understand session storage functions
- [ ] Update upload button to store file 
- [ ] Write function (window.sessionStorage) that can access stored file 
- [ ] Ensure file's data can be accessed

**Comment by shuai-yang** [comment]
Task1 - Research using session data is done (1.5 hours) :
Conclusion:

- We will use sessionStorage object to store key/value pairs in the browser. The sessionStorage object stores only one session.(The data is deleted when the browser is closed).

> Save Data to Session Storage
- sessionStorage.setItem("key", "value");
> Read Data from Session Storage
- let filename = sessionStorage.getItem("key");
> Remove Data from Session Storage
- sessionStorage.removeItem("key");
>  Remove All (Clear session Storage)
sessionStorage.clear();

**Comment by shuai-yang** [comment]
Pull Request  #148 is not approved because sample test file is not available so far and the expected result is unclear. See detail discussion in #148.

While we made progress on parser, we may modify this stories to make more sense. 

Note: spent 1.5 hours on research before starting coding and spent 4 hours coding and communicating with team.

**Comment by katerosenthal** [comment]
> Pull Request #148 is not approved because sample test file is not available so far and the expected result is unclear. See detail discussion in #148.
> 
> While we made progress on parser, we may modify this stories to make more sense.
> 
> Note: spent 1.5 hours on research before starting coding and spent 4 hours coding and communicating with team about this story.

There was clearly some confusion on what this story was and needed in order to be completed. Code submitted was a nearly exact duplicate of existing code and added no new functionality, so PR #148 was not approved and was closed. More details are provided in the PR.

We do have the necessary files from the sponsor via an email from September 16th.

Because this functionality was halfway/partially implemented while creating a parser (#144), the rest of it will be addressed within a similar parsing story instead and this story will be closed.
